### ISSUE TITLE
Could you help me to freeze the trained models?

### ISSUE THREAD

**Issue by newforrestgump001** [Stick-To/Object-Detection-API-Tensorflow#44]
I want to deploy the trained models in C+++, how to freeze them, could you give me some hints?
Thanks,
jack

**Comment by zcunyi** [comment]
> I want to deploy the trained models in C+++, how to freeze them, could you give me some hints?
> Thanks,
> jack

hi, have you trained on your data success ?
After sevel epochs my loss was always nan when I trained on my datasets.

**Comment by newforrestgump001** [comment]
@zcunyi ,sorry! not yet, I wonder how to freeze the chpt after training the arbitrary network.

**Comment by GallonDeng** [comment]
also want to know how to freeze the trained model
